nodejs/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+# fallback placeholder. This should come from the `nodejs-version-lookup` role to
+# determine the apps's preferred nodejs version.
+nodejs_version: 4.1.2
```

Comment:
So by having this as a default value. This will cause every roadrunner app to install node/npm. I think this is fine but just verifying with you and @orischwartz that we want to be installing node/npm for eFolder Express and Feedback as well.  

If we don't want to, then we could make the defaults here null, and only conditionally install node?